config/configmap/inferenceservice.yaml

```diff
@@ -24,8 +24,8 @@ data:
             "multiModelServer": "false"
         },
         "sklearn": {
-            "image": "gcr.io/kfserving/sklearnserver",
-            "defaultImageVersion": "v0.4.1",
+            "image": "docker.io/seldonio/mlserver",
+            "defaultImageVersion": "0.1.2",
```

Comment:
Let's still keep the default to old v1 version for a few releases and then we can start defaulting to mlserver once they migrate away from v1. With v1beta1 user can specify the image and version on the inference service spec itself if they want to use mlserver.